echidna/scripts/dump_smeared_energy.py

```diff
@@ -34,6 +34,9 @@ def main(args):
     Args:
       args (Namespace): Container for arguments. See
         >>> python dump_smeared_energy.py -h
```

Comment:
This code snippet isn't formatted correctly, I think it needs a blank line between and maybe `::` between `See` and the code block. Also `>>>` isn't really necessary as this would be run from the main terminal prompt not an interactive python session.